README.md

```diff
@@ -42,6 +43,15 @@ If not, copy the lines where the script failed into a
 [new GitHub Issue](https://github.com/thoughtbot/laptop/issues/new) for us.
 Or, attach the whole log file as an attachment.
 
+OS X El Capitan (10.11)
+-----------------------
+
+You may have problems installing Homebrew for the first time on OS X El
+Capitan with permissioning in `/usr`. See the 
```

Comment:
Perhaps "with permissioning in `/usr`" => "due to permission changes to the `/usr` folder (which is where Homebrew's installation folder is located)"